DIFFERENCES.md

```diff
@@ -7,3 +7,6 @@
 - API calls which pass JSON body as "query" (e.g. upgradeProduct() or
   deactivateProduct()) can include unexpected attributes (mostly bools) which
   don't support "omitempty" tag. API seems to ignore these correctly.
+- When proxy credentials are incorrect, go version returns different error
+  message than the original ruby one. Both are misleading and don't indicate
+  any proxy related problems.
```

Comment:
It depends on the proxy server. With squid:
`SUSEConnect error: Get "https://scc.suse.com/connect/systems/activations": Proxy Authentication Required`

Reply:
Lets keep it here for now. I think it's best to get the JSON decoding error implemented like in ruby version and get those things aligned.